bucket/Imagine.json

```
@@ -0,0 +1,46 @@
+{
+    "version": "1.1.4",
+    "description": "(1MB) Freeware Image & Animation Viewer for Windows",
+    "homepage": "https://www.nyam.pe.kr/dev/imagine/",
+    "license": "Donationware",

```

```suggestion
    "license": {
        "identifier": "Freeware",
        "url": "https://www.nyam.pe.kr/dev/imagine/"
    },
```
Just use `Freeware`. See [wiki/App Manifests/license](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests#required-properties) for details.